ci/infra/testrunner/tests/conftest.py

```diff
@@ -50,7 +50,7 @@ def deployment(request, bootstrap, skuba, kubectl):
     if request.config.getoption("skip_setup") != 'deployed':
         skuba.join_nodes()
 
-    wait(kubectl.run_kubectl, 'wait --timeout=1m --for=condition=Ready pods --all --namespace=kube-system', wait_delay=60, wait_timeout=300, wait_backoff=30, wait_retries=5)
+    wait(kubectl.run_kubectl, 'wait --timeout=1m --for=condition=Ready pods --all --namespace=kube-system', wait_delay=60, wait_timeout=300, wait_backoff=30, wait_retries=5, wait_allow=(RuntimeError))
```

Comment:
can we just use `kubectl wait` with a timeout of 5m here? i don't see the need to have a nested `wait` procedure with 2 different waiting mechanisms, it looks a bit overcomplex to me

Reply:
this was explained in https://github.com/SUSE/skuba/pull/768